region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -272,10 +272,24 @@
       "link": [1, 3],
       "name": "Morph Power Beam Sidehopper Kill",
       "requires": [
-        "Morph"
+        "Morph",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          {"and": [
+            "canDodgeWhileShooting",
+            {"enemyDamage": {
+              "enemy": "Sidehopper",
+              "type": "contact",
+              "hits": 2
+            }},
+          ]}
+        ]}
       ],
       "clearsObstacles": ["A"],
-      "note": "The Sidehoppers can't hit a morphed Samus, so they can be picked off damage-free."
+      "note": [
+        "The Sidehoppers can't hit a morphed Samus, making it possible to kill them damage-free.",
+        "Fight them one at a time from a corner with a quick unmorph-shoot-morph combo while holding angle up."
```

Comment:
i actually found it easier to fight them when not in a corner. get right under where they will jump from, and aim to where they will jump to and you can get several shots before remorphing